### ISSUE TITLE
Accessibility regression 1.82 > 1.83: Terminal buffer cannot be read with Shift+Tab

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#31867]
Type: <b>Bug</b>

There is an accessibility regression from 1.82 > 1.83 with the integrated terminal.

In 1.83, with JAWS
1. Run a command in integrated terminal
2. Press Shift+Tab

Actual: JAWS is silent. Experience is clunky, up/down seems to write commands in the terminal input. Much worse experience versus 1.82
Expected: Focus moves to the readonly edit where I can review output.

Note: Yes, Alt+F2 is a nice buffer, but Shift+Tab should take you to an accessible readonly edit like it did in 1.82

In 1.82, that would be:
1. Run ac ommand in integrated terminal
2. Press Shift+Tab

Actual: JAWS reads readonly edit and you can navigate up and down the terminal output. It's super nice, because you can use the shortcut Ctrl+` to move between the code and the output of the terminal. So I can write some code, run some lines with Shift+Enter (in Python), then press Ctrl+`and focus moves back to where I was in the terminal output and I can just read down, then back to code with Ctrl+`.

Please fix that regression. I tried other flows, like that focus virtual buffer whenever a command is run, which is good, and let's keep that option, but not as good as it was in 1.82 and I find no equivalent in 1.83

VS Code version: Code - Insiders 1.83.0-insider (4268e464763087044d0c1b5bdd37ebbe683cadfa, 2023-09-29T20:42:50.630Z)
OS version: Windows_NT x64 10.0.19045
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-9700F CPU - 3.00GHz (8 x 3000)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|15.94GB (6.97GB free)|
|Process Argv|--crash-reporter-id 00ed479f-54df-4c3b-bb92-bdf8893179a3|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (15)</summary>

Extension|Author (truncated)|Version
---|---|---
csdevkit|ms-|0.5.127
csharp|ms-|2.4.4
vscode-dotnet-runtime|ms-|1.8.0
vscodeintellicode-csharp|ms-|0.1.26
python|ms-|2023.17.12721006
vscode-pylance|ms-|2023.9.31
vsliveshare|ms-|1.0.5883
java|red|1.23.0
intellicode-api-usage-examples|Vis|0.2.8
vscodeintellicode|Vis|1.2.30
vscode-java-debug|vsc|0.54.0
vscode-java-dependency|vsc|0.23.1
vscode-java-pack|vsc|0.25.14
vscode-java-test|vsc|0.40.0
vscode-maven|vsc|0.42.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythontb:30258533
pythonptprofiler:30281269
vshan820:30294714
vscod805:30301674
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
pythonvs932:30404738
py29gd2263:30784851
vsclangdf:30492506
c4g48928:30535728
dsvsc012:30540252
pynewext54:30618038
a9j8j154:30646983
showlangstatbar:30737417
7ij38806:30687742
pythonfmttext:30716741
fixshowwlkth:30771523
showindicator:30805243
pythongtdpath:30726887
i26e3531:30792625
welcomedialog:30812478
pythonnosmt12:30779711
pythonidxpt:30768918
pythonnoceb:30776497
copilotsettingt:30808721
synctok:30821570
dsvsc013:30777762
dsvsc014:30777825
diffeditorv2:30786206
pythonmypyww:30841056
pythonmpsinfo:30842935
dsvsc015:30821418
pythontestfixtcf:30826907
pythonfb280951:30830809
pythonregdiag:30842812
pythonmypyd1:30850305
pythoncet0cf:30848248

```

</details>
